src/App.js

```diff
@@ -55,11 +55,20 @@ export class App extends Component {
     });
 
     let al = [...this.state.events];
-    al.push(data)
 
-    this.setState( {
+    if (data["data"] != null){
+      al.push(data);
+      this.setState( {
         events: al
     });
+    return;
+    } 
+    if (data["data"] === undefined){
+      alert("Event received with an invalid or missing data payload. Check the console for more information");
```

Comment:
maybe not an alert :D

Maybe we can use the alert component instead? https://material-ui.com/components/alert/

Reply:
yes of course!